src/ctapipe/image/extractor.py

```diff
@@ -1308,7 +1308,7 @@ def __call__(
         self, waveforms, tel_id, selected_gain_channel, broken_pixels
     ) -> DL1CameraContainer:
         container = DL1CameraContainer(
-            image=np.nanvar(waveforms, dtype="float32", axis=2),
+            image=np.nanvar(waveforms, dtype="float32", axis=2)[0],
```

Comment:
We shall be able to work with multiple gains, why do you introduce this change?

Reply:
np.nanvar was giving me an extra dimension on top of the gain. I think I can fix this with the keepdims parameter